### ISSUE TITLE
Void Thaumaturge Boots step ability stops working after travelling across dimensions

### ISSUE THREAD

**Issue by Vapaman** [TheCodex6824/ThaumicAugmentation#143]
Installed mods:
Baubles
Thaumcraft
Thaumic Augmentation 2.0.13

To reproduce: Equip charged boots. They work as intented, giving a speed boost and allowing you to step up 1 block. After travelling to any dimension its ability to move up 1 block stops working. I also noticed that the speed boost still works but acceleration while jumping is reduced. 

Workaround: Relogging fixes this issue until you travel to another dimension again.

**Comment by Vapaman** [comment]
I also noticed that taking the boots off and trying to jump while hugging the block seems cause the player to be repulsed from the block.

**Comment by DoomRater** [comment]
I have a dev build of 2.0.13 with some step up compatibility added and I do not notice this issue.  Of course I needed a compatibility toggle turned on due to using ArtemisLib mods like Gulliver Reborn and Resizing Potion, so I may be exploiting code that doesn't affect your instance.  I'll do another dev test with just base Thaumcraft and default config settings later.

TL;DR: This _might_ already be fixed in dev.

**Comment by TheCodex6824** [comment]
This is probably not fixed, at least not with the default config. I wasn't aware that the step height and such got reset on changing dimensions, so I'll add a fix for that.

**Comment by Determancer** [comment]
@TheCodex6824 Thaumostatic Harness has a similar problem, dimension changing by Waystones causes the flight stops working, but taking it off and wearing it again can restart it.

**Comment by TheCodex6824** [comment]
> Thaumostatic Harness has a similar problem, dimension changing by Waystones causes the flight stops working, but taking it off and wearing it again can restart it.

Looks like MC just resets all my stuff on dimension change, I'll have the next version just reapply everything to prevent more variants of this from popping up.

**Comment by TheCodex6824** [comment]
Original variant of this has been fixed. As for the waystones thing, I think that's an issue on their end, as with vanilla nether portals and dimensional fractures the harness works as expected and flight continues working.
